src/test/java/org/breedinginsight/api/auth/AuthServiceLoginHandlerUnitTest.java

```diff
@@ -106,42 +109,47 @@ public void returnsPassedUrlGoodHttpsUrl() {
         SimpleCookies cookies = new SimpleCookies(ConversionService.SHARED);
         cookies.put("redirect-login", returnUrlCookie);
         doReturn(cookies).when(request).getCookies();
+        doReturn(false).when(request).isSecure();
         AuthServiceLoginHandler authServiceSpy = spy(authServiceLoginHandler);
         when(authServiceSpy.getCurrentRequest()).thenReturn(Optional.of(request));
 
-        Cookie jwtToken = Cookie.of("phylo-token", "test");
-        List<Cookie> securityCookies = List.of(jwtToken);
+        UserDetails user = new UserDetails(userName, new ArrayList<>());
 
-        HttpResponse response = authServiceSpy.loginSuccessWithCookies(securityCookies);
+        HttpResponse response = authServiceSpy.loginSuccess(user, request);
 
         reset(request);
-        checkAssertions(response, jwtToken, expectedUrl);
+        checkAssertions(response, expectedUrl);
     }
 
+    // TODO: Fix this test
     @Test
     public void returnsDefaultUrlCookieNotExist() {
 
         HttpRequest request = mock(HttpRequest.class, CALLS_REAL_METHODS);
         SimpleCookies cookies = new SimpleCookies(ConversionService.SHARED);
         doReturn(cookies).when(request).getCookies();
+        doReturn(false).when(request).isSecure();
         AuthServiceLoginHandler authServiceSpy = spy(authServiceLoginHandler);
         when(authServiceSpy.getCurrentRequest()).thenReturn(Optional.of(request));
 
-        Cookie jwtToken = Cookie.of("phylo-token", "test");
-        List<Cookie> securityCookies = List.of(jwtToken);
+        UserDetails user = new UserDetails(userName, new ArrayList<>());
 
-        HttpResponse response = authServiceSpy.loginSuccessWithCookies(securityCookies);
+        HttpResponse response = authServiceSpy.loginSuccess(user, request);
 
         reset(request);
-        checkAssertions(response, jwtToken, defaultUrl);
+        checkAssertions(response, defaultUrl);
     }
 
-    public void checkAssertions(HttpResponse response, Cookie jwtToken, String expectedLocation) {
+    public void checkAssertions(HttpResponse response, String expectedLocation) {
 
         // Check location is as expected
         String redirectLocation = response.getHeaders().get("Location");
         assertEquals(expectedLocation, redirectLocation);
 
+        // Check that the jwt cookie was returned
+
```

Comment:
It seems there is code missing here?

Reply:
This was moved below to the end of the test. I moved this comment down there.